docs/compass/04-01-installation.md

```diff
@@ -0,0 +1,78 @@
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: api-packages-override
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    kyma-project.io/installation: ""
+data:
+  global.enableAPIPackages: "true"
+EOF
+```
+
+## Compass as a Central Management Plane
+
+This is a multi-cluster mode in which you need one cluster with Compass and at least one cluster with Kyma Runtime, which you can connect and manage using Compass. This mode allows you to integrate your Runtimes with Applications and manage them in one central place.
+
+![Kyma mode2](./assets/kyma-mode2.svg)
+
+
+### Kyma Compass
+
+This is a multi-tenant and multi-Runtime mode that provides a cluster with Compass and only the selected Kyma components that Compass uses. To enable this mode, create this ConfigMap and then perform the cluster Kyma installation using the
+ [`installer-cr-cluster-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-compass.yaml.tpl) configuration file:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: compass-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: compass
+    kyma-project.io/installation: ""
+data:
+  # The parameter that enables the Compass gateway, as the default Kyma gateway is disabled in this installation mode
+  gateway.gateway.enabled: "true"
+  # The name of the currently used gateway
+  global.istio.gateway.name: compass-istio-gateway
+  # The Namespace of the currently used gateway
+  global.istio.gateway.namespace: compass-system
+  # The parameter that disables preconfiguration for the Compass Runtime Agent
+  global.agentPreconfiguration: "false"
+  # The Namespace with a Secret that contains a certificate for the Connector Service
+  global.connector.secrets.ca.namespace: compass-system
+```
+
+### Kyma Runtime
+
+This is a single-tenant mode that provides complete cluster Kyma installation with the Runtime Agent. To enable this mode, follow the cluster Kyma installation using the [`installer-cr-cluster-runtime.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-runtime.yaml.tpl) configuration file. Before you start the installation, apply the following ConfigMap which enables the [API Packages](https://github.com/kyma-incubator/compass/blob/master/docs/compass/03-packages-api.md) functionality:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: api-packages-override
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    kyma-project.io/installation: ""
+data:
+  global.enableAPIPackages: "true"
```

Comment:
```suggestion
  global.disableLegacyConnectivity: "true"
```
This is outdated. The flag name has changed. See [this](https://github.com/kyma-project/kyma/pull/7876/files) PR.